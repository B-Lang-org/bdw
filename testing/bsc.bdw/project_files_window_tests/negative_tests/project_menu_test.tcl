
# Typechecks the current project
handlers::typecheck

# Compiles the current project
handlers::compile

# Refreshes the project hierarchy
handlers::refresh

# Creates a new project
handlers::open_project project.bspec 

# Typechecks the current project
handlers::typecheck

# Compiles the current project
handlers::compile

# Closes the project 
handlers::close_project

# Exits the worksptation
exit
