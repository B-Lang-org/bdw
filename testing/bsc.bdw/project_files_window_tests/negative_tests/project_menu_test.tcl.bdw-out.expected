There is no open project
There is no open project
There is no open project
The project top file is not specified.
The project top file is not specified.
